Workbooks/UpdateCompliance/AlertsTargetBuild/Alerts for Target build.workbook

```diff
@@ -0,0 +1,366 @@
+            "label": "workspace",
+            "type": 5,
+            "isRequired": true,
+            "query": "resources\r\n| where type == \"microsoft.operationalinsights/workspaces\"\r\n| where \"{mappedSubscription:subscriptionId}\" == subscriptionId\r\n| project id",
+            "crossComponentResources": [
+              "{mappedSubscription}"
+            ],
+            "value": "",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "03761a72-d969-426b-ba54-d1e709610b2d",
+            "version": "KqlParameterItem/1.0",
+            "name": "_WorkbookContext",
+            "label": "Workbook Context",
+            "type": 1,
+            "description": "Whether workbook is opened standalone or as part of the Hub.",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "criteriaContext": {
+                  "operator": "Default",
+                  "resultValType": "static",
+                  "resultVal": "Focused"
+                }
+              }
+            ],
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "f503a201-a9bd-4085-8862-22e2c0735ed4",
+            "version": "KqlParameterItem/1.0",
+            "name": "_SnapshotTime",
+            "label": "Snapshot time",
+            "type": 1,
+            "description": "Time indicating the latest snapshot",
+            "isRequired": true,
+            "query": "UCClient\r\n| where TimeGenerated >= ago(7d)\r\n| summarize max(TimeGenerated)",
+            "crossComponentResources": [
+              "{mappedWorkspace}"
+            ],
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "1b83a70d-303d-49fa-80d5-f36ac5af7ad6",
+            "version": "KqlParameterItem/1.0",
+            "name": "_TargetBuild",
+            "label": "Target Build",
+            "type": 1,
+            "description": "Set alerts for this Target Build",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "value": "10.0.19042.1766"
+          },
+          {
+            "version": "KqlParameterItem/1.0",
+            "name": "_OSVersion",
+            "label": "Target Version",
+            "type": 1,
+            "description": "Target OS Version",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "id": "48dc0814-6eb8-4083-bdcd-54caee036b9b",
+            "value": "Windows 10, version 20H2"
```

Comment:
just checking, these values will be updated manually if needed? is there an api you can call to get this value?